anima-jws-voucher.mkd

```diff
@@ -50,52 +50,45 @@ normative:
-
-In addition to explaining how the format is created, the "application/voucher-jws+json" media type is registered and examples are provided.
+In addition to specifying the format, the "application/voucher-jws+json" media type is registered and examples are provided.
 
 --- middle
 
 # Introduction
 
-"A Voucher Artifact for Bootstrapping Protocols" {{I-D.ietf-anima-rfc8366bis}} defines a YANG-based data structure used in "Bootstrapping Remote Secure Key Infrastructure" {{!BRSKI}} and
-"Secure Zero Touch Provisioning" {{?SZTP}} to transfer ownership of a device from a manufacturer to a new owner (customer or operational domain).
+"A Voucher Artifact for Bootstrapping Protocols" {{I-D.ietf-anima-rfc8366bis}} defines a YANG-based data structure
+used in "Bootstrapping Remote Secure Key Infrastructure" (BRSKI) {{!RFC8995}} and "Secure Zero Touch Provisioning" (SZTP) {{?RFC8572}}
+to transfer ownership of a device from a manufacturer to a new owner (customer or operational domain).
 That document provides a serialization of the voucher data to JSON {{RFC8259}} with cryptographic signing according to the Cryptographic Message Syntax (CMS) {{?RFC5652}}.
-The resulting voucher artifact has the media type "application/voucher-cms+json".
+That resulting voucher artifact has the media type `application/voucher-cms+json`.
 
-This document provides cryptographic signing of the JSON Voucher Data in form of JSON Web Signature (JWS) {{RFC7515}}
-and the media type "application/voucher-jws+json".
+This document provides cryptographic signing of voucher data in form of JSON Web Signature (JWS) {{RFC7515}} and the media type `application/voucher-jws+json` to identify the voucher format.
 The encoding specified in this document is used by {{?I-D.ietf-anima-brski-prm}}
 and may be more handy for use cases already using Javascript Object Signing and Encryption (JOSE).
-This document should be considered as enhancement of {{I-D.ietf-anima-rfc8366bis}},
-as it provides a new voucher form with media type "application/voucher-jws+json" and the related serialization.
-It does not extend the YANG definition of {{I-D.ietf-anima-rfc8366bis}}.
 
-Another example of an enhancement of {{I-D.ietf-anima-rfc8366bis}} is {{?I-D.ietf-anima-constrained-voucher}}.
-It provides the serialization of voucher data to CBOR {{?RFC8949}},
-with the signature in form of COSE {{?RFC8812}} and the media type "application/voucher-cose+cbor".
+This document should be considered as enhancement of {{I-D.ietf-anima-rfc8366bis}}, as it provides a new voucher format.
+It is similar to {{?I-D.ietf-anima-constrained-voucher}}, which provides cryptographic signing according COSE {{?RFC8812}} and the media type `application/voucher-cose+cbor`.
+These documents do not change nor extend the YANG definitions of {{I-D.ietf-anima-rfc8366bis}}.
```

Comment:
Attempt why it is its own draft and not part of 8366bis.

Reply:
Proposal looks good to me. The proposal from Michael regarding separation of CMS encoding may also make sense, but would somehow break with the RFC 8366 approach. Separating further formats as described also based on industrial use is good.